docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,108 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Nowra ADC    | | Nowra Tower  | 118.850         | NW_TWR        |
+| Nowra SMC   ||  Nowra Ground  | 135.850         | NW_GND        |
+| Nowra ACD    | | Nowra Delivery  | 128.350         | NW_DEL       |
+| Nowra ATIS    |   | | 125.65         | YSNW_ATIS       |
+
+
+## Airspace
+NWA ADC owns the Class C airspace from `SFC` to `A020`, 5nm from the NW TACAN. This airspace is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+
+## Aerodrome
+
+### General Guidelines
+
+1. **Ground Taxiing for Wheeled Helicopters**
+    - Where practicable, wheeled helicopters are encouraged to “ground taxi” on prepared surfaces to minimize rotor wash and its effects.
+
+### The Runway Mode options available are:
+
+a) Runway 21 and Runway 08;  
+b) Runway 21 and Runway 26;  
+c) Runway 21 only;  
+d) Runway 03 and Runway 08;  
+e) Runway 03 and Runway 26;   
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Miscellaneous
+### Circuit Procedures
+Circuit operations at Nowra occur within the CIRA, a `5NM` radius of NW TACAN, from the `SFC` to `A020`.
+
+- Fixed wing: `A020`
+- Rotary wing: `A012`
+- Flight Deck Procedural Trainer: not above `A007`
```

Comment:
What is a Flight Deck Procedural Trainer?

Reply:
i was asking @mattkelly4 that earlier, similar to a helicopter, listed in the NW ERSA and military SOP

Reply:
It's a replica of a ship's landing deck, allows visual approaches by day and night apparently. Located south of the runway intersection
![Uploading image.png…]()